### ISSUE TITLE
Google preview button should be non-clickable sticker

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#234]
The Google Preview image is meant to meet Google's branding requirements - it is the sticker only, not a clickable button. 
https://developers.google.com/books/branding
http://www.google.com/intl/en/googlebooks/images/gbs_preview_sticker1.png

[Currently the Preview button is clickable, which means the link is represented twice in the page = confusing for accessibility. Also the button cannot be explicit about full text vs preview so it's been reported as a usability issue.]

![screen shot 2014-06-05 at 5 44 19 pm](https://cloud.githubusercontent.com/assets/6945691/3195569/b86df1e8-ed13-11e3-8859-02731fcbf12f.png)

![screen shot 2014-06-05 at 5 40 26 pm](https://cloud.githubusercontent.com/assets/6945691/3195566/a33ce360-ed13-11e3-9c0b-eee142c9dbc7.png)
